keystone/server/flask/core.py

```diff
@@ -79,9 +79,20 @@
                       'config_file': '/etc/keystone/watcher.yaml',
                       'include_initiator_user_id_in_metric': 'true',
                       'include_target_project_id_in_metric': 'false'}),
+    # CCloud: add rate_limit middleware
+    _Middleware(namespace='rate_limit.middleware',
+                ep='rate_limit',
```

Comment:
the endpoint must be `rate-limit` see https://github.com/sapcc/openstack-rate-limit-middleware/blob/master/setup.cfg#L33-L39